README.md

```diff
@@ -68,8 +92,12 @@ Meet with your corresponding group again. And, again, select which set will be D
 
 ***Q1.7 Briefly summarize your findings as a Designer.  Is the customer happy with your work?  How could it be further improved?***
 
+Customer seems to be happy overall, their idea was similar to our idea. It could be further improved by making it even more attractive to users.
+
 ***Q1.8 Briefly summarize your findings as a Customer.  Will the proposed solution meet your needs?  Was it substantially different than what you may have initially envisioned?***
 
+the solution was an improvement over our proposed solution. However, we felt that the application was limited in its use because it had to partner with individual websites for the application to work perfectly.
```

Comment:
That's a good point about partnering with individual websites. Like the example you gave above, if we wanted to ban a certain social media site until after a job application was passed in, you might have to check in with the company to make sure that they are ok connecting their job application form to this blocker application you've developed.